src/fittable_models.jl

```diff
@@ -0,0 +1,40 @@
+# Convenience function for wavelength conversion
+aa_to_invum(wave::Real) = 10000 / wave
+aa_to_invum(wave::Quantity) = aa_to_invum(ustrip(u"angstrom", wave))
+
+"""
+    FM90(λ::Real, C1 = 0.10, C2 = 0.70, C3 = 3.23, C4 = 0.41, xo = 4.60, γ = 0.9)
+    FM90(λ::Quantity, C1 = 0.10, C2 = 0.70, C3 = 3.23, C4 = 0.41, xo = 4.60, γ = 0.9)
+
+The parameters C1, C2, C3, C4, xo and gamma are y-intercept, slope, bump amplitude, FUV rise amplitude,
+bump centroid and bump width respectively.
+
+Fitzpatrick & Massa (1990) 6 parameter ultraviolet shape model, this model is only applicable at UV wavelengths.
+
+If `λ` is a `Unitful.Quantity` it will be automatically converted to Å and the
+returned value will be `UnitfulAstro.mag`.
+"""
+function FM90(λ::Real, C1 = 0.10, C2 = 0.70, C3 = 3.23, C4 = 0.41, xo = 4.60, γ = 0.9)
```

Comment:
@giordano @mileslucas What would be your suggestion regarding these fittable models? Should I go for the current implemented version, or should I make `FM90` a struct and rewrite the function like `(z::FM90)(x::Real)`?

Reply:
This is actually a pretty big design choice. It makes sense to have a struct if a user fits it, then can evaluate many times without having to re-fit it. I would take that approach here, personally. 

To keep continuity, though, this implies we might want to change how the non-parametric models are presented. 

@giordano what do you think about having the following system?
- `struct LAW end`
- `(::LAW)(wave)`
- `function law() end`

where the last one is just a convenient alias?

Reply:
Sounds good, let's see how it works

Reply:
okay, I can follow the previous code up in a separate PR after this one to unify the interface. Will be a good opportunity to restructure some stuff, now that I have an extra ~6 months of Julia under my belt.